src/transformers/modeling_tf_bert.py

```diff
@@ -843,17 +890,44 @@ def call(self, inputs, **kwargs):
             >>> prediction_scores, seq_relationship_scores = outputs[:2]
 
         """
-        return_dict = kwargs.get("return_dict")
         return_dict = return_dict if return_dict is not None else self.bert.return_dict
-        outputs = self.bert(inputs, **kwargs)
+
+        if isinstance(inputs, (tuple, list)):
+            labels = inputs[9] if len(inputs) > 9 else labels
+            next_sentence_label = inputs[10] if len(inputs) > 10 else next_sentence_label
+            if len(inputs) > 9:
+                inputs = inputs[:9]
+        elif isinstance(inputs, (dict, BatchEncoding)):
+            labels = inputs.pop("labels", labels)
+            next_sentence_label = inputs.pop("next_sentence_label", next_sentence_label)
```

Comment:
Just realized that this seems to be an issue in all TF models: if `return_dict` is defined in the `inputs` (they're either a tuple, a list or a dict), the value of `return_dict` won't be used here?

Reply:
Yep! But will be solved with the new input parsing. A fix will be in a PR that will arrive soon.